DNN Platform/Library/Common/Globals.cs

```diff
@@ -1,3842 +1,3935 @@
+        public static string GetShortDate(string strDate)
+        {
+            if (!string.IsNullOrEmpty(strDate))
+            {
+                DateTime datDate = Convert.ToDateTime(strDate);
+                string strYear = datDate.Year.ToString();
+                string strMonth = datDate.Month.ToString();
+                string strDay = datDate.Day.ToString();
+                strDate = strMonth + "/" + strDay + "/" + strYear;
+            }
+
+            return strDate;
+        }
+
+        /// <summary>
+        /// Determines whether current request contains admin control information.
+        /// </summary>
+        /// <returns>
+        ///   <c>true</c> if current request contains admin control information; otherwise, <c>false</c>.
+        /// </returns>
+        public static bool IsAdminControl()
+        {
+            // This is needed to avoid an exception if there is no Context.  This will occur if code is called from the Scheduler
+            if (HttpContext.Current == null)
+            {
+                return false;
+            }
+
+            return (!string.IsNullOrEmpty(HttpContext.Current.Request.QueryString["mid"])) || (!string.IsNullOrEmpty(HttpContext.Current.Request.QueryString["ctl"]));
+        }
+
+        /// <summary>
+        /// Determines whether current request use admin skin.
+        /// </summary>
+        /// <returns>
+        ///   <c>true</c> if current request use admin skin; otherwise, <c>false</c>.
+        /// </returns>
+        public static bool IsAdminSkin()
+        {
+            bool isAdminSkin = Null.NullBoolean;
+            if (HttpContext.Current != null)
+            {
+                string adminKeys = "tab,module,importmodule,exportmodule,help";
+                string controlKey = string.Empty;
+                if (HttpContext.Current.Request.QueryString["ctl"] != null)
+                {
+                    controlKey = HttpContext.Current.Request.QueryString["ctl"].ToLowerInvariant();
+                }
+
+                int moduleID = -1;
+                if (HttpContext.Current.Request.QueryString["mid"] != null)
+                {
+                    int.TryParse(HttpContext.Current.Request.QueryString["mid"], out moduleID);
+                }
+
+                isAdminSkin = (!string.IsNullOrEmpty(controlKey) && controlKey != "view" && moduleID != -1) ||
+                               (!string.IsNullOrEmpty(controlKey) && adminKeys.IndexOf(controlKey) != -1 && moduleID == -1);
+            }
+
+            return isAdminSkin;
+        }
+
+        /// <summary>
+        /// Returns whether the current tab is in EditMode.
+        /// </summary>
+        /// <returns><c>true</c> if the tab is in Edit mode; otherwise <c>false</c>.</returns>
+        public static bool IsEditMode()
+        {
+            // TODO: Refactor this as per decision on https://github.com/dnnsoftware/Dnn.Platform/issues/4213
```

Comment:
This TODO has been handled, right?
```suggestion
```

Reply:
Good point, addressed in https://github.com/dnnsoftware/Dnn.Platform/pull/4215/commits/a07f391daef00f485e63d786f0e27e48587027b5